DataTransferObject/UserDataTransferObject.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\DataTransferObject;
+
+use SumoCoders\FrameworkMultiUserBundle\User\User;
+
+interface UserDataTransferObject
+{
+    public static function fromUser(User $user);
+    public function getEntity();
```

Comment:
well, the `getEntity` makes an object based on some parameters, which makes it a factory method.